sim/core/sim.go

```diff
@@ -126,9 +145,15 @@ func (sim *Simulation) reset() {
 		sim.Log("SIM RESET")
 		sim.Log("----------------------")
 	}
-	variation := sim.DurationVariation * 2
 
-	sim.Duration = sim.BaseDuration + time.Duration((sim.RandomFloat("sim duration") * float64(variation))) - sim.DurationVariation
+	// Reset primary targets damage taken for tracking health fights.
+	sim.Encounter.Targets[0].DamageTaken = 0
```

Comment:
Wouldn't the more intuitive approach be to stop at the sum of health from all targets, rather than only the primary target's health?

Reply:
I thought if this as "fight ends when boss dies" and the boss would be the 'primary target'. Killing "adds" probably doesn't end the encounter.

Reply:
I think that's a bit too literal. A very simple example of where this wouldn't work is simulating a trash pack, e.g. with several mobs that all need to be killed. In boss encounters the mobs are often killed too.

Reply:
Alright,I think I can agree with this :)